### PR TITLE
Add sourceHtml and rename source to sourceUri in WebView bindings

### PR DESCRIPTION
**Pull request by kristinns**
Renames `WebView.source` method to `WebView.sourceUri` and adds missing `WebView.sourceHtml`

### CODE REVIEW

**Comment by wokalski** [comment]
@kristinns would using `bs.deriving abstract` work here? It's a bit clearer to read.

**Comment by kristinns** [comment]
@wokalski that would be great but I'm not sure how I could do that.
Mainly because `WebView.sourceUri` and `WebView.sourceHtml` create different JS types but we are telling the reason compiler that this is the same type.
Do you know how this could be done with `bs.deriving abstract`?

File: src/components/webView.rei [hunk 1]
```diff
@@ -1,8 +1,11 @@
 type source;
 
-let source:
```
Comment (wokalski) [inline src/components/webView.rei]: This is a breaking change. We can make it non breaking by aliasing `source` to `sourceUri` and marking it as deprecated

Comment (kristinns) [reply src/components/webView.rei]: Good point, fixed in dec94b7

File: src/components/webView.re [hunk 2]
```diff
@@ -26,6 +26,19 @@ external source :
...
+external sourceHtml :
+  (
+    ~html: string=?,
+    ~baseUrl: string=?,
+    unit
+  ) =>
+  source =
+  "";
+
+[@deprecated "Please use WebView.sourceUri instead"]
```
Comment (wokalski) [inline src/components/webView.re]: This is only needed in the `rei`. Also did u test it with the RNTester to verify? I never trust reason deprecated annotations :P

Comment (kristinns) [reply src/components/webView.re]: Ah ok, fixed in dec94b7 (amended to the commit)
And yes the deprecation worked, got warning message
`deprecated: BsReactNative.WebView.source, Please use WebView.sourceUri instead`

**Pull request merged**